src/main/java/com/google/api/codegen/viewmodel/PageStreamingDescriptorClassView.java

```diff
@@ -45,6 +45,10 @@
 
   public abstract boolean resourcesFieldIsMap();
 
+  public boolean resourcesFieldAsEntrySet() {
+    return resourceTypeName() != null && resourceTypeName().contains("<");
```

Comment:
- Nit: Did you mean `IsEntrySet`? When a reader sees "as," they may expect this method to convert `resourcesField` into an entry set.
- Consider adding a comment with a sample `resourceTypeName` value.

Reply:
Agree, no idea why I called that `resourceFieldAsEntrySet` instead of `resourceFieldIsEntrySet`. Renamed.

Reply:
This still says `As`

Reply:
Opps, forgot to push =) Thanks!